includes/admin/post-types/post-tables/class-llms-admin-post-table-achievements.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * LLMS_Admin_Post_Table_Achievement class
+ *
+ * @package LifterLMS/Admin/PostTypes/PostTables/Classes
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+// TODO: remove this when the new loader will be implemented.
+require_once LLMS_PLUGIN_DIR . '/includes/traits/llms-trait-certificate-achievement-template-post-list-table.php';
+
+/**
+ * Customize display of the achievement post table.
+ *
+ * @since [version]
+ */
+class LLMS_Admin_Post_Table_Achievements {
+
+	use LLMS_Trait_Certificate_Achievement_Templates_Post_List_Table;
```

Comment:
I don't like this classname, I think I'd prefer:

```suggestion
	use LLMS_Trait_Award_Templates_Post_List_Table;
```